src/main/java/io/kontur/disasterninja/client/InsightsLLMApiClient.java

```diff
@@ -34,4 +35,12 @@ public LLMAnalyticsDto getLLMAnalytics(AnalyticsRequestDto llmAnalyticsRequest)
                         });
         return response.getBody();
     }
+
+    public SearchDto search(UUID appId, String query) {
+        ResponseEntity<SearchDto> response = llmAnalyticsRestTemplate
+                .exchange(String.format("/search?appId=%s&query=%s", appId, query), GET, httpEntityWithUserBearerAuthIfPresent(null),
+                        new ParameterizedTypeReference<>() {
+                        });
```

Comment:
Maybe
```suggestion
                .exchange(String.format("/search?appId=%s&query=%s", appId, query), GET,
                        httpEntityWithUserBearerAuthIfPresent(null),
                        new ParameterizedTypeReference<>() {});
```
for an easier read?